airbyte-commons-server/src/test/java/io/airbyte/commons/server/handlers/OAuthHandlerTest.java

```diff
@@ -48,7 +50,8 @@ public void init() {
     trackingClient = mock(TrackingClient.class);
     httpClient = Mockito.mock(HttpClient.class);
     secretsRepositoryReader = mock(SecretsRepositoryReader.class);
-    handler = new OAuthHandler(configRepository, httpClient, trackingClient, secretsRepositoryReader);
+    secretsRepositoryWriter = mock(SecretsRepositoryWriter.class);
+    handler = new OAuthHandler(configRepository, httpClient, trackingClient, secretsRepositoryReader, secretsRepositoryWriter);
```

Comment:
are we going to add some test cases around this change?

Reply:
Yeah planning to add some as soon as we solidify on what else needs to be added to the PR.